elide-spring/elide-spring-boot-autoconfigure/src/main/java/com/yahoo/elide/spring/config/ElideAutoConfiguration.java

```diff
@@ -40,8 +49,32 @@
  */
 @Configuration
 @EnableConfigurationProperties(ElideConfigProperties.class)
+@Slf4j
 public class ElideAutoConfiguration {
 
+    /**
+     * Creates a entity compiler for compiling dynamic config classes.
+     * @param settings
+     * @return An instance of ElideDynamicEntityCompiler
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    public ElideDynamicEntityCompiler buildElideDynamicEntityCompiler(ElideConfigProperties settings) {
+
+        ElideDynamicEntityCompiler compiler = null;
+
+        if (settings.getDynamicConfig().isEnabled()) {
+            compiler = new ElideDynamicEntityCompiler(settings.getDynamicConfig().getPath());
+            try {
+                compiler.compile(settings.getDynamicConfig().getPath());
+            } catch (Exception e) {
+                e.printStackTrace();
```

Comment:
Don't print stack traces - log the exception.  Also - don't catch Exception.  Just let it propagate.